doc/api/errors.md

```diff
@@ -344,7 +344,10 @@ The location information will be one of:
 * `plain-filename.js:line:column`, if the frame represents a call internal
   to Node.js.
 * `/absolute/path/to/file.js:line:column`, if the frame represents a call in
-  a user program, or its dependencies.
+  a user program (using commonJS module system), or its dependencies.
+* `<transport-protocol>:///file/url/path/to/file.mjs:line:column`, if the frame
```

Comment:
It's not a file url path if it's not using the `file:` protocol (it could be `data:`, and `https:` support is also being discussed).
```suggestion
* `<transport-protocol>://url/to/module:line:column`, if the frame
```

Reply:
Fixed